org.guerinoni.qTsConverter.appdata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>org.guerinoni.qTsConverter</id>
+    <name>qTsConverter</name>
+    <summary>A simple tool to convert qt translation file (ts) to other format (xlsx / csv) and vice versa.</summary>
+    <description>
```

Comment:
This file doesn't seem to be used here and it should be upstream as it's not a flatpak/flathub specific thing, it's a freedesktop standard that Flathub enforces.